iDiff/utils/output_templates/multiVersionOutput.txt

```diff
@@ -1,6 +1,10 @@
-Packages found only in {{.Diff.Image1}}:{{range $name, $value := .Diff.Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{end}}{{"\n"}}
-Packages found only in {{.Diff.Image2}}:{{range $name, $value := .Diff.Packages2}}{{"\n"}}{{print "-"}}{{$name}}{{end}}
+-----{{.DiffType}}-----
 
-Version differences:{{"\n"}}	(Package:	{{.Diff.Image1}}{{"\t\t"}}{{.Diff.Image2}}){{range .Diff.InfoDiff}}
-	{{.Package}}:	{{range .Info1}}{{.Version}}{{end}}	{{"\t"}} {{range .Info2}}{{.Version}}{{end}}
-	{{end}}
+Packages found only in {{.Diff.Image1}}:{{ if not .Diff.Packages1 }} No unique packages in Image1{{ else }}{{range $name, $value := .Diff.Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{end}}{{ end }}
```

Comment:
@nkubala hmm what command did you run that it gave you pythonlatest as a name?  

also, the reason the name in the image is shortened like that is because we use that field as the name to save and access the temporary image directory that's built, so it can't have the slashes and colon...  but I can look into changing that